src/client/components/Card/Card.styles.css

```diff
@@ -0,0 +1,24 @@
+.title-text-card {
+  box-sizing: border-box;
+  position: absolute;
+  border: 1px solid #d5d5d5;
+  border-radius: 15px;
+  background-color: #f6f6f6;
+  color: #666666;
+  overflow: auto;
+  font-family: Roboto, sans-serif;
+  width: 282px;
```

Comment:
Not sure that we want to specify exact width. Are you sure? Have you tested on all screen sizes? Remove and we will later add it if needed.